src/main/java/com/labsynch/labseer/service/ExportServiceImpl.java

```diff
@@ -172,6 +172,7 @@ private int writeLotsToSDF(String exportSDFFileName, Collection<LotDTO> lotDTOs)
 			if (lotDTO.getParentIsMixture() != null) mol.setProperty("Parent Is Mixture", lotDTO.getParentIsMixture().toString());
 			exporter.writeMol(mol);
 		}
+		exporter.close();
```

Comment:
The other implementations had `close` but never used it.  The Rdkit mol writer requires a close to actually write the contents so I added this here.  The others still work with or without the close.